packages/wonder-blocks-dropdown/components/multi-select.md

```diff
@@ -159,25 +160,24 @@ class SimpleMultiSelect extends React.Component {
 
     render() {
         return <MultiSelect
-            items={[
-                {label: "Stark", value: "1"},
-                {label: "Arryn", value: "2"},
-                {label: "Baratheon", value: "3"},
-                {label: "Tully", value: "4"},
-                {label: "Greyjoy", value: "5"},
-                {label: "Lannister", value: "6"},
-                {label: "Tyrell", value: "7"},
-                {label: "Martell", value: "8"},
-                {label: "Targaryen", value: "9"},
-            ]}
             onChange={(selectedValues) => this.handleChanges(selectedValues)}
             selectedValues={this.state.selectedValues}
             selectItemType="Great Houses"
-            style={{
-                width: 170,
-                maxWidth: 170,
-            }}
-        />;
+            openerStyle={styles.setWidth}
+            dropdownStyle={styles.setWidth}
```

Comment:
I don't think we should ever be setting the width on the dropdown.

Reply:
Done! Wow, that was much simpler than what I was anticipating having to do and also makes a whole lot more sense 😅